### PR TITLE
switch compass for bourbon

### PR DESCRIPTION
**Pull request by vietnogi**
ping @indytechcook & @tladendo

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -55,15 +56,16 @@ gulp.task('plato', function (done) {
...
  */
 gulp.task('scss', function () {
   log('Compiling SCSS --> CSS');
 
-  return gulp
-    .src(config.scss)
-    .pipe($.compass(config.compass))
+  return gulp.src(config.scss)
+    .pipe(sass({
+      includePaths: require('node-bourbon').includePaths
```
Comment (indytechcook) [inline gulpfile.js]: @vietnogi where are these include paths set?  Does it pull from config.scss?

**Comment by indytechcook** [comment]
lgtm

File: gulpfile.js (hunk 1 above)
Comment (indytechcook) [inline gulpfile.js]: I did a quick search and indeed they do.

Comment (tladendo) [inline gulpfile.js]: Why the inline `require`?

Comment (indytechcook) [inline gulpfile.js]: Looking at the docs, it's looks like that's the recommended way:  https://www.npmjs.com/package/node-bourbon.  The include paths are pulled from the stream so perhaps is needs to be required at that point.

Comment (tladendo) [inline gulpfile.js]: Hm, not sure I follow, but it's very minor anyway.

**Comment by tladendo** [comment]
lgtm

**Pull request merged**